crates/sui-benchmark/src/drivers/bench_driver.rs

```diff
@@ -241,8 +242,7 @@ async fn ctrl_c() -> std::io::Result<()> {
 impl Driver<(BenchmarkStats, StressStats)> for BenchDriver {
     async fn run(
         &self,
-        workloads: Vec<WorkloadInfo>,
-        proxy: Arc<dyn ValidatorProxy + Sync + Send>,
+        proxy_workloads: Vec<(Arc<dyn ValidatorProxy + Send + Sync>, Vec<WorkloadInfo>)>,
```

Comment:
@arun-koshy - It seems a much easier approach could be to have a proxy which can round robin to several full nodes behind the scene. I don't think we need to have a workload per full node in that scenario.

Reply:
Again we don't look like we will need scaling with fullnodes for the initial benchmarking, but without separating the workload we will have to wait for local execution at each full node to execute the subsequent transactions which would limit our throughput.

Reply:
Again we don't look like we will need scaling with fullnodes for the initial benchmarking, but without separating the workload we will have to wait for local execution at each full node to execute the subsequent transactions which would limit our throughput.